src/webgpu/api/validation/encoding/queries/general.spec.ts

```diff
@@ -5,10 +5,76 @@ TODO:
     - queryIndex {in, out of} range for GPUQuerySet
     - GPUQuerySet {valid, invalid}
         - or {undefined}, for occlusionQuerySet
-    - ?
+    - x = {occlusion, pipeline statistics, timestamp} query
```

Comment:
I just realized that this PR leaves this TODO in here, but I think most or all of it is done. Can you please update the TODO? It's really important for us to track what work remains. Sorry for missing this the first time. @haoxli